docs/pages/api-docs/breadcrumbs.md

```diff
@@ -28,9 +28,9 @@ The `MuiBreadcrumbs` name can be used for providing [default props](/customizati
 
 | Name | Type | Default | Description |
 |:-----|:-----|:--------|:------------|
-| <span class="prop-name required">children&nbsp;*</span> | <span class="prop-type">node</span> |  | The breadcrumb children. |
+| <span class="prop-name">children</span> | <span class="prop-type">node</span> |  | The breadcrumb children. |
 | <span class="prop-name">classes</span> | <span class="prop-type">object</span> |  | Override or extend the styles applied to the component. See [CSS API](#css) below for more details. |
-| <span class="prop-name">component</span> | <span class="prop-type">elementType</span> | <span class="prop-default">'nav'</span> | The component used for the root node. Either a string to use a HTML element or a component. By default, it maps the variant to a good default headline component. |
```

Comment:
Would require a lot of type gymnastics to get this back as far as I can tell. I don't think this documentation adds anything because it's implicit: Obviously we want to choose a "good default". The question is what this default is and we have that already answered in the default column. Then you ask: "But why this default value?" and "Because it's good" is not a meaningful answer.